php/class-wp-seo-settings.php

```diff
@@ -321,6 +321,15 @@ public function has_term_fields( $taxonomy ) {
 	}
 
 	/**
+	 * Helper to see whether the 'archive' to 'taxonomy' migration should run.
+	 *
+	 * @return bool
+	 */
+	public function should_taxonomy_migration_run() {
+		return apply_filters( 'wp_seo_taxonomy_migration_opt_in', false );
```

Comment:
This filter [needs docs](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters). Also, it seems like the hook name could be based on the method name? Doesn't have to, just one fewer name in the mix.